readme.md

```diff
@@ -41,6 +41,7 @@ Once installation is working, try creating and requesting resources. Note that t
 * **Question:** What is the difference between mongoose and mongodb?
 
 * **Your Answer:**
+Mongo DB is a document based, cloud based database. mongoose is a is an object relational mapper which is the interface used to work with Mongo DB.
```

Comment:
MongoDB doesn't _have_ to be cloud-based. It can also be installed locally on your machine.